features/backend/shipping_categories.feature

```diff
@@ -7,9 +7,9 @@ Feature: Shipping categories
     Background:
         Given store has default configuration
           And there are following shipping categories:
-            | name    |
-            | Regular |
-            | Heavy   |
+            | name    | code |
+            | Regular | SC1  |
+            | Heavy   | SC2  |
```

Comment:
It would be nice to have some way to autogenerate codes. If we don't use codes in further scenarios, they shouldn't be required while creating them in background.